home-page.html

```diff
@@ -0,0 +1,190 @@
+                </li>
+                <li class="nav-branch">
+                    <div>
+                        <span class="nav-links">INTERNAL PARTS</span>
+                        <ul class="sub-navbar">
+                            <div class="sub-nav-container">
+                                <li><span class="sub-nav-links">Gearboxes</span></li>
+                                <li><span class="sub-nav-links">Motors</span></li>
+                            </div>
+                            
+                        </ul>
+                    </div>
+                </li>
+            </ul>
+        </div>
+        <div>
+            <h1 class="catalog-title">
+                CATALOG
+            </h1>
+            <div class="home-page-content">
+                <div class="home-product">
+                    <img class="img-product" src="images/product/ak101-img.jpeg" alt=" an ak101 ">
+                    <h5 class="product-title">
+                        AK-101 
+                    </h5>
+                    <div class="description-container">
+                        <p class="product-description">very good rifle i'm speechless </p>
+                    </div>
+                    <div class="product-stat">
+                        <button class="product-add-button">ADD</button>
+                        <button class="product-plus">+</button>
+                        <p class="product-number">1</p>
+                        <button class="product-minus">-</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        
+
+        <div class="home-footer">
+            <!-TODO: still need to write the something in the footer ->
+            <h2 class="footer-title">TEST</h2>
+            <div class="footer-contact">
+                <!-TODO: still need to write the definition for the info in the footer ->
+                <p>INFO INFO</p>
+                <p>ALTRA INFO</p>    
+            </div>
+            <div class="footer-img">
+                <span><img class="social-imges" src="images/home-page/instagram-img.png" alt=" the instagram logo"></span>
+                <span><img class="social-imges" src="images/home-page/facebook-img.png" alt=" the facebook logo"></span>
+                <span><img class="social-imges" src="images/home-page/whatsapp-img.png" alt=" the whatsapp logo"></span>
+                <span><img class="social-imges" src="images/home-page/x-img.png" alt=" the twitter logo"></span>
+            </div>
+            
+        </div>
+
+        <script type="text/javascript">
+
+            window.onload = function (){
+
+                let itemCount = 1;
+
+                const buttonMinus = document.querySelector(".product-minus");
+                const buttonPlus = document.querySelector(".product-plus");
+                const incrementItemCount = document.querySelector(".product-number");
+
+                buttonMinus.addEventListener("click", decrement);
```

Comment:
You don't need the reference of the element. You can get it and directly assign the event

Reply:
fixed